shared/cluster.go

```diff
@@ -357,30 +357,26 @@ func parsePods(res string) []Pod {
 	pods := make([]Pod, 0, 10)
 	podList := strings.Split(strings.TrimSpace(res), "\n")
 
-	offset := 1
 	for _, rec := range podList {
+		offset := 0
 		fields := strings.Fields(rec)
 		if strings.TrimSpace(rec) == "" || len(fields) < 9 {
 			continue
 		}
-		var p *Pod
-		if len(fields) == 9 {
-			offset = offset - 1
-		} else {
-			p = &Pod{
-				NameSpace: fields[offset],
-			}
-		}
-		p = &Pod{
-			Name:      fields[offset],
-			Ready:     fields[offset + 1],
-			Status:    fields[offset + 2],
-			Restarts:  fields[offset + 3],
-			Age:       fields[offset + 4],
-			NodeIP:    fields[offset + 5],
-			Node:      fields[offset + 6],
+		var p Pod
+		if len(fields) != 9 {
```

Comment:
if len(fields) == 8 {
	      pod.NameSpace = fields[0]
	   	offset = 1
 }

Reply:
I saw this, made the change, and didn't commit it 🤦